### ISSUE TITLE
[Feature Request] Add directly copied images

### ISSUE THREAD

**Issue by LukeHuckman** [Tudmotu/gnome-shell-extension-clipboard-indicator#261]
It would be nice to have directly copied images (e.g. right clicking and choosing "copy image" in web browsers or copy a selected portion of an image in GIMP) added to the clipboard history.

**Comment by khinbaptista** [comment]
omg yes
I just installed it a few days ago, all the reviews were super positive, and I was so disappointed that I cannot copy image data anymore. Using `gnome-screenshot --area --clipboard` it simply doesn't copy anything (and I use it a lot, so I will need to keep the extension disabled until it is fixed/implemented.

**Comment by lordnodd** [comment]
I use flameshot for that purpose, its great if you are maintaining notes and even for random things I found it quite handy. It also gives an option clipboard, I wish we can get it done on this repository. This repo is close to Ditto, clipboard manager that I used on windows except for this feature. I am learning how it can be made possible here, so if no one does it I'll take some time and make a pull request XD.
